.github/workflows/scorecard.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
     steps:
       - name: "Checkout code"
-        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
```

Comment:
Verified this is the hash for v3.6.0:
https://github.com/actions/checkout/commit/f43a0e5ff2bd294095638e18286ca9a3d1956744